### PR TITLE
Fix missing PTU for missing certificate

### PR DESCRIPTION
**Pull request by tkireva**

### CODE REVIEW

**Comment by tkireva** [comment]
Please review:
@okozlovlux @LitvinenkoIra

File: src/components/security_manager/src/crypto_manager_impl.cc [hunk 1]
```diff
@@ -234,7 +234,7 @@ bool CryptoManagerImpl::OnCertificateUpdated(const std::string& data) {
 }
 
 SSLContext* CryptoManagerImpl::CreateSSLContext() {
-  if (NULL == context_ || IsCertificateUpdateRequired()) {
+  if (NULL == context_) {
```
Comment (LevchenkoS) [inline src/components/security_manager/src/crypto_manager_impl.cc]: @tkireva, why you remove `IsCertificateUpdateRequired()` from check?

Comment (tkireva) [inline src/components/security_manager/src/crypto_manager_impl.cc]: @LevchenkoS  @okozlovlux If IsCertificateUpdateRequired() is not removed here, PTU for missing certificate or for expired certificate won't be triggered - protocol handler will send startServiceACK/NACK. Currently I am working on a fix that in this situation will trigger PTU, wait PTU to be finished and after that will sent startServiceACK/NACK.

Comment (tkireva) [inline src/components/security_manager/src/crypto_manager_impl.cc]: Currently the main problem is that the SDL should wait PTU finished before send any answer.

**Pull request merged**